atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -142,6 +153,22 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji connection established")
 
+    def _fetch_log_files(self):
+        try:
+            namespace = get_build_json().get('metadata', {}).get('namespace', None)
+        except KeyError:
+            namespace = None
+        osbs_conf = Configuration(conf_file=None, openshift_uri=self.url,
+                                  use_auth=self.use_auth, verify_ssl=self.verify_ssl,
+                                  namespace=namespace)
+        osbs = OSBS(osbs_conf, osbs_conf)
+        res = self.workflow.prebuild_results.get(PLUGIN_KOJI_PARENT_KEY) or {}
+        build_info = res.get('parent-image-koji-build') or {}
```

Comment:
We want the OpenShift build-id of the currently running build (`get_build_json()['metadata']['name']`), not the Koji build ID of parent image.

Reply:
Fixed.